tap_adroll/client.py

```diff
@@ -73,10 +75,11 @@ def _make_request(self, method, endpoint, headers=None, params=None, data=None,
         )
 
         # TODO: We should merge headers with some default headers like user_agent
-        response = self.session.request(method, full_url, headers=headers, params=params, data=data)
+        headers = headers or {}
+        headers['Authorization'] = f"Bearer {self.access_token}"
+        response = requests.request(method, full_url, headers=headers, params=params, data=data)
```

Comment:
This needs to be reverted. Accidental commit from our token refresh code changes.